apps/frontend/src/libs/modules/api/base-http-api.ts

```diff
@@ -81,6 +215,17 @@ class BaseHTTPApi implements HTTPApi {
 		return headers;
 	}
 
+	private getUrl(path: string, query?: Record<string, string>): string {
```

Comment:
```suggestion
	private getUrl<T extends Record<string, unknown>>(path: string, query?: T): string {
```

Reply:
I'm getting "Type parameter T is used only once in the function signature." now. So I guess I'll just use Record<string, unknown> as a type for query, it'll work the same.

Reply:
I guess I'll just use Record<string, unknown> type for query, because linter does not like that we use generic only in method signature, and it would work the same.